index.js

```diff
@@ -16,7 +15,8 @@ module.exports = options => {
 
   const load = (config) => {
     config.routes.forEach((route) => {
-      app.use(router(route, config));
+      config.route = route;
```

Comment:
if `config.route` is temporary and used only by the current route, perhaps we could write `const routeConfig = Object.assign({route}, config)` and pass that as an argument to the route.